resources/devices/device_profile.json

```diff
@@ -0,0 +1,353 @@
+            "text": "Sensor - Weather",
+            "risk": "Limited",
+            "id": 24
+          },
+          {
+            "text": "Sensor - Steam",
+            "risk": "High",
+            "id": 25
+          },
+          {
+            "text": "Sensor - Air Flow",
+            "risk": "Limited",
+            "id": 26
+          },
+          {
+            "text": "Sensor - Lighting",
+            "risk": "Limited",
+            "id": 27
+          },
+          {
+            "text": "Sensor - Other",
+            "risk": "High",
+            "id": 28
+          },
+          {
+            "text": "Sensor - Air Quality",
+            "risk": "Limited",
+            "id": 29
+          },
+          {
+            "text": "Monitoring - Fire System",
+            "risk": "Limited",
+            "id": 30
+          },
+          {
+            "text": "Monitoring - Emergency Lighting",
+            "risk": "Limited",
+            "id": 31
+          },
+          {
+            "text": "Monitoring - Other",
+            "risk": "High",
+            "id": 32
+          },
+          {
+            "text": "Monitoring - UPS",
+            "risk": "Limited",
+            "id": 33
+          },
+          {
+            "text": "Meter - Water",
+            "risk": "Limited",
+            "id": 34
+          },
+          {
+            "text": "Meter - Gas",
+            "risk": "Limited",
+            "id": 35
+          },
+          {
+            "text": "Meter - Electricity",
+            "risk": "Limited",
+            "id": 36
+          },
+          {
+            "text": "Meter - Other",
+            "risk": "High",
+            "id": 37
+          },
+          {
+            "text": "Other",
+            "risk": "High",
+            "id": 38
+          },
+          {
+            "text": "Data - Storage",
+            "risk": "High",
+            "id": 39
+          },
+          {
+            "text": "Data - Processing",
+            "risk": "High",
+            "id": 40
+          },
+          {
+            "text": "Tablet",
+            "risk": "High",
+            "id": 41
+          }
+        ]
+      },
+      {
+        "id": 2,
+        "question": "Does your device process any sensitive information? ",
+        "validation": {
+          "required": true
+        },
+        "type": "select",
+        "options": [
+          {
+            "id": 1,
+            "text": "Yes",
+            "risk": "Limited"
+          },
+          {
+            "id": 2,
+            "text": "No",
+            "risk": "High"
+          },
+          {
+            "id": 3,
+            "text": "I don't know",
+            "risk": "High"
+          }
+        ]
+      },
+      {
+        "id": 3,
+        "question": "Please select the technology this device falls into",
+        "validation": {
+          "required": true
+        },
+        "type": "select",
+        "options": [
+          {
+            "id": 1,
```

Comment:
Previously, for strings and select answers, the string has been passed on its own so I'm not sure of the need for an ID here